tests/conftest.py

```diff
@@ -1,13 +1,11 @@
-import multiprocessing
-
 import pytest
 from app.create_app import create_app
 from selenium import webdriver
 from selenium.webdriver.chrome.options import Options
 from selenium.webdriver.chrome.service import Service
 from webdriver_manager.chrome import ChromeDriverManager
 
-multiprocessing.set_start_method("fork")  # Required on macOSX
+# multiprocessing.set_start_method("fork")  # Required on macOSX
```

Comment:
Might be good to do a conditional here rather than commenting it out

Maybe 

```python3
if platform.system() == "Darwin":
    multiprocessing.set_start_method("fork")  # Required on macOSX
```